frontend/src/app/office-hours/section/section-creation-form/section-creation-form.component.html

```diff
@@ -0,0 +1,43 @@
+<form class="page-container" [formGroup]="sectionForm" (ngSubmit)="onSubmit()">
+  <mat-card appearance="outlined">
+    <div class="card-container">
+      <mat-card-title>Create Office Hours Section</mat-card-title>
+    </div>
+    <mat-divider></mat-divider>
+    <mat-card-subtitle class="card-container">
+      Office Hours Sections allow for students to check into office hours for
+      your courses. 'Course Name' is what will appear to students in your
+      classe(s). Select all course sections you wish to use this Office Hours
+      section for.
+    </mat-card-subtitle>
+    <mat-card-content class="card-container">
+      <p>Course Name:</p>
+      <mat-form-field>
+        <mat-label>Ex. COMP 110: Introduction to Programming</mat-label>
+        <input matInput required formControlName="section_name" />
+      </mat-form-field>
+      <p>Course Section(s):</p>
+      <mat-form-field>
+        <mat-label>Select all courses:</mat-label>
+        <mat-select formControlName="academic_sections" multiple required>
+          <mat-option
+            *ngFor="let section of academicSections"
+            [value]="section.id">
+            {{ section.course_id }}-{{ section.number }}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+      <div class="buttons">
+        <button
+          mat-flat-button
```

Comment:
Can use outlined style, but I forget how to apply this - look at the buttons in the organization feature! (Button conventions are something we will specifically revisit in the redesign)